module-services/service-cellular/ServiceCellular.cpp

```diff
@@ -2241,11 +2217,15 @@ auto ServiceCellular::handleCellularCallerIdNotification(sys::Message *msg) -> s
         return std::make_shared<CellularResponseMessage>(hangUpCallBusy());
     }
 
-    if (!callManager.isCallerInfoComplete()) {
-        bus.sendMulticast(std::make_shared<CellularCallerIdMessage>(message->getNubmer()),
-                          sys::BusChannel::ServiceCellularNotifications);
-        callManager.completeCallerInfo();
+    // TODO
+    // 1. move this multicast to ongoingCall
+    // 2. send it only once per clip not multiple times
+    // 3. check how otehr services handle this caller info complete
```

Comment:
https://github.com/mudita/MuditaOS/blob/master/doc/MuditaCppCodingGuidelines.md
M-9 :trollface:

Reply:
this is still a draft ;)